offers/README.md

```diff
@@ -1,234 +1,295 @@
 # Offers
 
-The Offer object holds the details about a card linked offer. Offers can be created by developers using the [Offer API](https://reference.fidel.uk/v1/reference#create-offer) or by Brands/Merchants using the CLO dashboard at [clo.fidel.uk](https://clo.fidel.uk).
-A card linked offer specifies a set of parameters that will be used to qualify a card transaction made at a participating Brand’s online or offline store.
+Fidel Offers help you create and manage card-linked offers with various retailers – all in one place. Developers can create Offers via the [Fidel Offers API](https://reference.fidel.uk/v1/reference#create-offer), which allows your application to create and update Offers, link and unlink Locations to the created Offers, activate and deactivate Offers on specific Cards and send the Offers for approval to a Brand. Brands and Merchants can interact with the Fidel Offers by using the [Fidel CLO Dashboard]((https://clo.fidel.uk)).
 
-## Offer object
+## Offer Object
+The Offers API's central piece of data is the Fidel Offer object, which holds all the details about a card-linked Offer. The card-linked offer has a set of parameters used to qualify any [card transaction](https://fidel.uk/docs/transactions) made at a participating Brands' linked Location.
 
-Once you have created an offer, you can request the contents of the object using the [Get Offer](https://reference.fidel.uk/reference#get-offer) API call. The response will appear similar to the following:
+The Offer object looks similar to the following and will be returned by the API in a multitude of situations. Transaction objects will also have a smaller version of the object inside, making it easier to retrieve the full Offer object if necessary.
```

Comment:
is it worth saying `Qualified transaction objects will(...)`

Reply:
From my understanding, all transaction objects will have the offer property. The ones that didn't qualify will have a rejection message, and the qualified ones will have a performanceFee property.

Reply:
Correct, so the offer will appear in the transactions if the transaction is from a location linked to an offer. If the transaction is done on a location that isn't linked to an offer, the transaction.offer = null. But I think the way @AlexLakatos described it's good.

Reply:
fine for me, sounds good